src/Apps/Collect/CollectApp.tsx

```diff
@@ -36,6 +49,16 @@ export class CollectApp extends Component<CollectAppProps> {
         <Meta property="og:description" content={description} />
         <Meta property="twitter:description" content={description} />
         <Link rel="canonical" href={canonicalHref} />
+        <BreadCrumbList
+          items={[
+            { path: "/collect", name: "Collect" },
+            mediumMeta && {
+              path: `/collect/${mediumMeta.id}`,
+              name: mediumMeta.name,
+            },
```

Comment:
It is common to include meta tags for SEO in the components that represent a page, and having to import `sharify` for every `< BreadCrumbList>` seems like an opportunity for consolidation. This would keep high-level components cleaner because environment variables won't be spread out. Though I also see why the direct import from `react-head` with a string interpolation with `APP_URL` makes sense. The two items are required to handle two different routes:

 * When it routes to `/collect`:
    * The `items` props will be `[{ path: "/collect", name: "Collect" }]`
    * So Google will show `Artsy > Collect`
 * When it routes to e.g. `/collect/painting`:
    * The `items` props will be `[{ path: "/collect", name: "Collect" }, { path: "/collect/panting", name: "Painting" }]`
    * So Google will show `Artsy > Collect > Painting`

I think it makes more sense that mediums belong to `Collect` rather than the top-level `Artsy` since they are options on the Collect page. The top-level `Artsy` link is added automatically by the `<BreadCrumbList>` component.

Reply:
Yeah I understand the need for meta tag data but I guess I don't fully understand why you have `mediumMeta` and check for its presence versus checking for the presence of `medium` and do something like `/collect/${medium}`. We could probably even refactor `canonicalHref` to `canonicalPath`and just have the path (i.e. `/collect/${medium}` or `/collect`) then on line 50 do something like `sd.APP_URL + canonicalPath`. I was just thinking that lines 31-38 seem redundant and wasn't sure if I was missing something or the reason why they are needed. 

Regarding the routing, `canonicalHref` currently handles resolving to whatever path was routed to based on whether the medium parameter exists, but again maybe there is something I'm not considering or don't understand. Perhaps we can chat about it later.

Reply:
I used the `mediumMeta.name` that comes from the GraphQL query (the `aggregations` field holds mediums) so we don't have to worry about updating the breadcrumb list when we add/update/remove mediums on the backend side. This also aligns with how the medium checkboxes are rendered and reduces the cost of manual work that needs to be done on the front end side. The downside is, as you pointed out, there would be a bit more, possibly duplicate, logic that finds and checks the existence of the medium specified.

I've added a new field `breadcrumbTitle` to `CollectMediumMetadata.ts` to remove the logic for now. These titles aren't the exact same but should be good enough to represent breadcrumb list data.